src/app/app.module.ts

```diff
@@ -4,10 +4,12 @@ import { BrowserModule } from '@angular/platform-browser';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { SharedModule } from './shared/shared.module';
+import { TasksComponent } from './app/tasks/tasks.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    TasksComponent
```

Comment:
Create a Task Module and Task Routing Module, then register this here.
In addition, go ahead and create another component under the task folder, called task-card. We can use the Tasks component to display a list of tasks.